main.php

```diff
@@ -10,18 +10,24 @@
 add_action('admin_enqueue_scripts', 'rg4wp_admin_styles');
 add_action('template_redirect', 'rg4wp_404_handler');
 add_action('wp_enqueue_script', 'load_jquery');
+add_action('plugins_loaded', 'rg4wp_checkUser');
+
+function rg4wp_isIgnoredDomain(): bool {
+    $domains = array_map('trim', explode(',', get_option('rg4wp_ignoredomains', '')));
+    return array_key_exists('SERVER_NAME', $_SERVER) && in_array($_SERVER['SERVER_NAME'], $domains);
+}
 
 if (
     (1 == get_option('rg4wp_js') || 1 == get_option('rg4wp_pulse'))
     && get_option('rg4wp_apikey')
     && !rg4wp_isIgnoredDomain()
-    && !is_admin()
+    && !(1 == get_option('rg4wp_noadmintracking', 0) && is_admin())
```

Comment:
Why did this change from not admin to is admin perms? Presumably, this is an admin only task, eh?